cpp/src/arrow/python/datetime.cc

```diff
@@ -233,25 +222,24 @@ static inline Status PyDate_convert_int(int64_t val, const DateUnit unit, int64_
   return Status::OK();
 }
 
-static inline Status PyTime_from_int(int64_t val, const TimeUnit::type unit,
-                                     PyObject** out) {
+Status PyTime_from_int(int64_t val, const TimeUnit::type unit, PyObject** out) {
   int64_t hour = 0, minute = 0, second = 0, microsecond = 0;
   RETURN_NOT_OK(PyTime_convert_int(val, unit, &hour, &minute, &second, &microsecond));
   *out = PyTime_FromTime(static_cast<int32_t>(hour), static_cast<int32_t>(minute),
                          static_cast<int32_t>(second), static_cast<int32_t>(microsecond));
   return Status::OK();
 }
 
-static inline Status PyDate_from_int(int64_t val, const DateUnit unit, PyObject** out) {
+Status PyDate_from_int(int64_t val, const DateUnit unit, PyObject** out) {
   int64_t year = 0, month = 0, day = 0;
   RETURN_NOT_OK(PyDate_convert_int(val, unit, &year, &month, &day));
   *out = PyDate_FromDate(static_cast<int32_t>(year), static_cast<int32_t>(month),
                          static_cast<int32_t>(day));
   return Status::OK();
 }
 
-static inline Status PyDateTime_from_int(int64_t val, const TimeUnit::type unit,
-                                         PyObject** out) {
+Status PyDateTime_from_int(int64_t val, const TimeUnit::type unit, PyObject** out) {
+  PyDateTime_IMPORT;
```

Comment:
@pitrou I'm not exactly sure where should I put this

Reply:
You mean the import?

Reply:
Yes.